docs/modules/ROOT/pages/how-tos/access-argo-cd.adoc

```diff
@@ -0,0 +1,17 @@
+= Access Argo CD running in a cluster
+
+Make sure your `kubectl` context is set to the cluster you want to work with.
+
+. Retrieve the admin secret
++
+----
+kubectl -n syn get secret steward -o json | jq -r .data.token | base64 --decode
+----
+. Port-forward to the Argo CD service
++
+----
+kubectl -n syn port-forward svc/argocd-server 8443:443
```

Comment:
I'm generally setting up the port-forward to port 80 of the service:

```suggestion
kubectl -n syn port-forward svc/argocd-server 8080:80
```
I'm not sure this makes a difference, so I've got no hard feelings either way here.

Reply:
I think that https/8443 leads to browser warnings?
I'd favor 8080:80, removes 2 clicks to ignore certificate warning

Reply:
Actually there is no browser warning because we're using plain-old-http. Nevertheless, I've changed the port-mapping as it makes much more sense.